### PR TITLE
RC - Active-Active GA 

### PR DESCRIPTION
**Pull request by lanceleonard**

### CODE REVIEW

File: content/rc/subscriptions/create-active-active-subscription.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Use the **VPC configuration** section of the **Advanced options** to define unique address blocks for each region.
+
+{{<image filename="images/rc/create-sub-active-active-cidr.png" width="75%" alt="Each region needs a unique CIDR address block to communicate securely with other instances." >}}{{< /image >}}
+
+When all **Deployment CIDR** regions display a green checkmark, you're ready to continue.  
+
+{{<image filename="images/rc/icon-cidr-address-ok.png" width="30px" alt="Greem chackmarks indicate valid CIDR address values." >}}{{< /image >}}
+
+Red exclamation marks indicate error conditions; the tool tip provides additional details.
```
Comment (rrelledge) [inline content/rc/subscriptions/create-active-active-subscription.md:58]:
```suggestion
Red exclamation marks indicate error conditions; the tooltip provides additional details.
```

File: content/rc/subscriptions/create-active-active-subscription.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+{{<image filename="images/rc/icon-cidr-address-error.png" width="30px" alt="Red exclamation points indicate CIDR address problems." >}}{{< /image >}}
+
+
+## Define throughput
+
+Each Active-Active instance coordinates changes with every other instance, which increases memory use and throughput.
+
+When create an Active-Active database, you define the throughput for each instance.
```
Comment (rrelledge) [inline content/rc/subscriptions/create-active-active-subscription.md:67]:
```suggestion
When you create an Active-Active database, you define the throughput for each instance.  
```

**Review by rrelledge — APPROVED** [review]
Looks good to me.

File: content/rc/subscriptions/create-active-active-subscription.md (hunk 2 above)
Comment (lanceleonard) [reply content/rc/subscriptions/create-active-active-subscription.md:67]: Fixed.

File: content/rc/subscriptions/create-active-active-subscription.md (hunk 1 above)
Comment (lanceleonard) [reply content/rc/subscriptions/create-active-active-subscription.md:58]: Fixed.

**Pull request merged**